sdk/iOS/test/MSSyncTableTests.m

```diff
@@ -1584,6 +1584,142 @@ -(void) testPullWithFetchLimitNotMultipleOfFifty
                   @"Invalid URL: %@", thirdRequest.URL.absoluteString);
 }
 
+-(void) testPullTable_UsePullSettings_NegativePageSize
```

Comment:
We need at least one test confirming page size is flowing into the inc sync test cases.